subprojects/docs/src/docs/userguide/jvm/java_testing.adoc

```diff
@@ -348,6 +348,32 @@ If `scanForTestClasses` is false and no include or exclude patterns are specifie
 With http://junit.org/junit5/docs/current/user-guide[JUnit Platform], only `includes` and `excludes` are used to filter test classes — `scanForTestClasses` has no effect.
 ====
 
+[[sec:test_logging]]
+== Test logging
+
+Gradle allows fine-tuned control over events that are logged to the console.  Logging is configurable on a per-log-level basis and by default, the following events are logged:
```

Comment:
Can we put each sentence in a single line?
```suggestion
Gradle allows fine-tuned control over events that are logged to the console.  
Logging is configurable on a per-log-level basis and by default, the following events are logged:
```

Reply:
Thanks for catching this.  I wish we had some way to enforce this in the build.  I try to keep this in mind when I'm initially writing the docs, but inevitably I make edits and forget.